File: c/1-sequence-struct/number-entered/main.c
// Write a program that prompts for a number and displays the message "The number you entered was [number]."

#include <stdio.h>
#include <stdlib.h>

int main()
{

    int number;
    printf("Enter a number: ");
    scanf("%d", &number);
    printf("\nThe number you entered was %d.\n", number);
    return 0;
}

File: java/1-sequence-struct/number-entered/src/Main.java
// Write a program that prompts for a number and displays the message "The number you entered was [number]."

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        // Instantiating object in java for data entry
        Scanner sc = new Scanner(System.in);

        int number;
        System.out.print("Enter a number: ");
        number = sc.nextInt();
        System.out.println("\nThe number you entered was " + number);

    }
}
